cross/x86/toolchain.cmake

```diff
@@ -7,6 +7,7 @@ set(CMAKE_SYSTEM_PROCESSOR i686)
 add_compile_options("-m32")
 add_compile_options("--sysroot=${CROSS_ROOTFS}")
 add_compile_options("-Wno-error=unused-command-line-argument")
+add_compile_options("-Wno-missing-prototype-for-cc")
```

Comment:
I have googled for the error. It looks like the compiler requires the function prototypes with no parameters to use "void" in the parents. Could you please try to add that to see if it silents the warnings?